composer.json

```diff
@@ -22,9 +22,6 @@
         "php": ">=7.0",
         "ext-mbstring": "*"
     },
-    "conflict": {
-        "twig/twig": ">=1,<1.38.1 || >=2,<2.12.1"
```

Comment:
Since `twig/twig` is not in the `require` section, shouldn't this be something like this (in `conflict`)?
```
"twig/twig": "<2.12.1"
```

And I just realised I should've done the same for Symfony < 3.4, I'll make a PR

Reply:
Yes, thanks